### PR TITLE
fix Display wrong epoch on Keras resume training

### PR DESCRIPTION
**Pull request by javoweb**
This fix `TqdmCallback` showing wrong information about epochs when resume training with `initial_epoch != 0`  as was reported on #1138 

The test has been provided but also the Colab example provided by @fabiocarrara  [here](https://colab.research.google.com/drive/1WMKllAs_edpU_kAV9klYk7q1lsP9DXwm?usp=sharing):

### CODE REVIEW

File: tqdm/keras.py [hunk 1]
```diff
@@ -69,10 +69,12 @@ def __init__(self, epochs=None, data_size=None, batch_size=None, verbose=1,
...
         params = self.params.get
         auto_total = params('epochs', params('nb_epoch', None))
-        if auto_total is not None:
+        if auto_total is not None and auto_total != self.epoch_bar.total:
             self.epoch_bar.reset(total=auto_total)
 
-    def on_epoch_begin(self, *_, **__):
+    def on_epoch_begin(self, epoch, *_, **__):
+        if self.epoch_bar.n < epoch:
+            self.epoch_bar.update(epoch-self.epoch_bar.n)
```
Comment (casperdcl) [inline tqdm/keras.py:77]: Is this only ever triggered exactly once when training is resumed? Or are there any other circumstances?

File: tqdm/keras.py [hunk 2]
```diff
@@ -69,10 +69,12 @@ def __init__(self, epochs=None, data_size=None, batch_size=None, verbose=1,
     def on_train_begin(self, *_, **__):
         params = self.params.get
         auto_total = params('epochs', params('nb_epoch', None))
-        if auto_total is not None:
+        if auto_total is not None and auto_total != self.epoch_bar.total:
```
Comment (casperdcl) [inline tqdm/keras.py:72]: why is this needed?

File: tqdm/keras.py (hunk 1 above)
Comment (javoweb) [reply tqdm/keras.py:77]: Yes, exactly once. I don't figure out other circumstances.

File: tqdm/keras.py (hunk 2 above)
Comment (javoweb) [reply tqdm/keras.py:72]: To avoid re-assigning the total when it is provided manually by the user.
